core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -353,6 +353,22 @@ a.expandbutton {
   width: 180px;
 }
 
+.executor-jvm-om-off-heap-memory-checkbox-div {
```

Comment:
how about using on class for all these optional checkbox div?

Reply:
> how about using on class for all these optional checkbox div?

on -> one ?

Reply:
yes